packages/Webkul/Shop/src/Http/Resources/ProductResource.php

```diff
@@ -50,7 +50,9 @@ public function toArray($request)
             'price_html'  => $productTypeInstance->getPriceHtml(),
             'ratings'     => [
                 'average' => $this->reviewHelper->getAverageRating($this),
-                'total'   => $this->reviewHelper->getTotalRating($this),
+                'total'   => core()->getConfigData('catalog.products.review.summary') == 'count_of_stars'
+                            ? $this->reviewHelper->getTotalRating($this)
+                            : $this->reviewHelper->getTotalReviews($this),
```

Comment:
No need of check here... simply provide one more array with total counts... i.e.

'reviews' => ['total' => ...]